coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove this file. It should be in the gitignore on master now too, so it should not continue to be added.